### PR TITLE
convert to log routing

### PR DESCRIPTION
**Pull request by natebrennand**
ticket: https://clever.atlassian.net/browse/DIS2-688

This PR is to convert these two log routing rules to application log routing: https://github.com/Clever/log-router/blob/88d69a7e48334628af3be9562047268d744b3d80/config/data.toml#L5-L27

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-	"gopkg.in/Clever/kayvee-go.v3"
+	"gopkg.in/Clever/kayvee-go.v6/logger"
+)
+
+const (
+	pollInterval = time.Minute
+)
+
+var (
+	lg = logger.New("gear-load-logger")
```
Comment (johnhuangclever) [inline main.go]: Can we make this `gearman-load-logger`?

File: main.go [hunk 2]
```diff
@@ -20,51 +25,41 @@ func logMetrics(g gearadmin.GearmanAdmin) {
...
+	gHost, err := discovery.Host("gearmand", "tcp")
+	if err == nil {
+		log.Fatalf("failed to get gearmand host: %s", err)
 	}
 
-	if h == "" {
-		log.Println("must specify host")
-		flag.PrintDefaults()
-		os.Exit(1)
+	gPort, err := discovery.Port("gearmand", "tcp")
```
Comment (nathanleiby) [inline main.go:48]: 👏

**Review by johnhuangclever — APPROVED** [review]

**Comment by johnhuangclever** [comment]
```
                                                                 ╖╖╖╖µ          
                                                                 ╙╢▒▓▓▓▒╦       
        ╓╦╣▒▓▒Θ⌐                                             ╔µ   ╠▒▓▓▓▓▓▓╕     
      ╔╣▒▓▓▓▓▌    ,                                         ╢╣░    ╚▒▓▓▓▓▓▓Q    
     ╢▒▓▓▓▓▓▒╜   ╠▒╦                                       -╫▒▓▒µ  '╢▓▓▓▓▓▓▓▌µ  
   ╓╣▒▓▓▓▓▓▒░   ╢▒▓▓╕                                      ░╫▒▓▓▒⌐  ]▒▓▓▓▓▓▓▓▒░ 
  '╢▓▓▓▓▓▓▓░   ╢▒▓▓▓░                                      ░╫▓▓▓▓▒µ ]▒▓▓▓▓▓▓▓▒░ 
  ¿╢▓▓▓▓▓▓▓▌⌐.╢▒▓▓▓▓░                                       ║▒▓▓▓▓▌░]▒▓▓▓▓▓▓▓Θ░ 
   ╠▓▓▓▓▓▓▓▌╓╣▓▓▓▓▓▒                                        "╫▓▓▓▓▓▌╠▒▓▓▓▓▓▓▒░  
   ╠▓▓▓▓▓▓▓▓╣▓▓▓▓▓▒'           -╖╖-        -╖╥╥╖⌐            ░▒▓▓▓▓▓▓▓▓▓▓▓▓▓▌   
   ]╫▓▓▓▓▓▓▓▓▓▓▓▓▒░          -'░╫▓▓▓╣░   -░░░▒▓▓▓╦            ╙▒▓▓▓▓▓▓▓▓▓▓▓▓▓░  
   ░▒╣▒▓▓▓▓▓▓▓▓▓▒░          ]╣╣▒▓▓▓▓▓▒░  ╠▒▒▓▓▓▓▓▓░            ╠▒▒▒▓▓▓▓▓▓▓▓▓▓░  
   ░▒▒▓▓▓▓▓▓▓▓▓▓Θ           ]▒▓▓▓▓▓▓▓Θ░  ░▒▓▓▓▓▓▓▒░            ]▒▒▓▓▓▓▓▓▓▓▓▓▌░  
    ░▒▓▓▓▓▓▓▓▓▓▒░            ╙Θ▒▓▓▓█Θ░    ╙Θ▒▓▓█Θ░░            '╠▒▓▓▓▓▓▓▓▓▓▓░   
     ╠▒▓▓▓▓▓▓▓▓Θ⌐             ░░░░░╥░⌐     '░╖╖╥╥░              ╠▒▓▓▓▓▓▓▓▓▓░    
      ╚▒▓▓▓▓▓▓▓░               ╙╢▒▓╣░▄╖╖╥╥▄-╓▒▓▒░              !╠▒▓▓▓▓▓▓▓█╨     
       ░▒▒▓▓▓▓▓▒⌐              ╔╣▓▓▓▓▒▒▒▓▓▓▓▓▓▓▓╦µ             '╢▒▒▓▓▓▓▓Ö       
        ╙δ▒▓▓▓▀             ╥╣▒▓▓▓▓▓▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▄╦            ╙▒▓▓▓Θ╜        
          ╢▓▓▌           ╔╣▒▓▓▒▓▓▓▓▓▓▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▄µ         .╫▓▓Θ          
          ╙╢▓▓▒╕       ╓╣▓▓▒▒▒ΘΘΘδδδÜδÉδΘΘδδδΘΘΘΘ▒▒▓▓▓▓▓▄µ      ,╫▓▓▒⌐          
           ╙╢▓▓▓Q,   ╓╣▓▓▓▒░░          '░⌐ ⌐⌐░░░░░░░╢▓▓▓▓▓▄   ,╥▒▓▓▒░           
             ╙Θ▓▓▓▒╦╗▒▓▓▓▒▒▒▒▒▄╦╤╦╖╖-╖╖░░░░░░░╣╣╣╣╣▒▓▓▓▓▓▓▓▓╦╣▒▓▓Θ╜             
               '╙Θ▒▓▓▓▓▓▓▓▓▒▒▒▒▒▓▓▓▒▒▒▒▒▒▒▒▒▒▓▓▓▓▒▒▒▒▓▓▓▓▓▓▓▓▓█▀                
                 .╠▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░╢▒▓▓▓▓▓▓▓░                 
                 ╠▓▓▓▓▓▓▒░░░░░░░░░░░░░░░░░░░░░░░░╢░░░░░╢▒▓▓▓▓▓▌░                
                -░ΘΘΘΘΘ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░╢╢▒███▒░░--             
         ░░░░░░░░░░░░░░░░░░░░░░░░░░░░L░G░T░M░░░╣░░░░░░░░░░░░░░░░░░░'  ░⌐        
        '░░░░""""╚╢▒╣╣╣╣╣Θ░░░░╢╣╣░░░░░░░░░░░░░╢╣╣░░░░░░░╣╣╦╣╣╣░      '░░░⌐      
      '░░░░       ░░░░░░░░░░░░░░░░░░░░░░░░░░░░╢░░░░░░░░░░░░░░░░░      ░░░░⌐     
      ░░░░░  ░░░░░░░░░░░░░░░░░░╢░░░╢░░░░░░░░╣╣╣╣╣░░░░░░░░░░░░░░░' ░⌐  ░░░░░     
     ░░░░░  ░░░"""'   ░░░░░░╦╣╣░░░░░░░░░░░░╣╣╣╣╣╣╣╣╣░░░░░░░░      "░░ ░░░╣░⌐    
    '░░░░ -░░░    -░░╙╨░'  "╙ÉÉδ░░δ░░░░░░░░δΘΘΘΘΘΘΘΘΘÉ░    '░░⌐    ░░░ ░╢╣░░    
    ░░░░  ░░░    ░░'                                         ░░░   '░░⌐ ░╢░░    
     ░░   ░░    ░░                                            "░░   "░░  '░░    
    !░░  '░⌐    '                                                    ░░  '░░    
     ╙░                                                                  ░░     
```

**Pull request merged**